src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -85,7 +96,11 @@ def run_relevance_eval(
         either booleans or None in the case where the LLM output could not be
         parsed.
     """
-
+    if not output_map:
+        output_map = {
+            RAG_RELEVANCY_PROMPT_RAILS_MAP[0]: True,  # "relevant":True
+            RAG_RELEVANCY_PROMPT_RAILS_MAP[1]: False,  # "irrelevant":False
+        }
```

Comment:
The rails are ordered dicts for a reason - this seems like a strange re-mapping.